src/pool.c

```diff
@@ -2977,6 +3034,22 @@ miner_on_login(json_object *message, client_t *client)
         }
     }
 
+    if (json_object_object_get_ex(params, "rigid", &rig_id))
+    {
+        if (!json_object_is_type(rig_id, json_type_string))
+            log_warn("rigid not a json_type_string");
+        else
+        {
+            const char *rigstr = json_object_get_string(rig_id);
+            strncpy(client->rig_id, rigstr, MAX_RIG_ID);
+            log_trace("Miner set rigid: %s", client->rig_id);
+        }
+    }
+    else
+    {
+        client->rig_id[0] = 0;
+    }
```

Comment:
This else not needed.

Reply:
removed https://github.com/jtgrassie/monero-pool/pull/81/commits/5cde6e295a98a8b1498673ba2ed098e4f7ec9583